src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -448,6 +448,7 @@ class TaskForm extends React.Component<Props, State> {
                                 'data-testid': 'task-form-date-input',
                             }}
                             isDisabled={isForbiddenErrorOnEdit}
+                            isKeyboardInputAllowed
```

Comment:
should this attribute have a type?
since it is being received as a prop then maybe in https://github.com/box/box-ui-elements/blob/fd9790216e6812d009a2d6bc8120afd526403464/src/elements/content-sidebar/activity-feed/task-form/TaskForm.js#L45

Reply:
yup! the type is defined here: https://github.com/box/box-ui-elements/pull/2682/files#diff-8c22974687f38296f0e4c9b61ba406c1a06dd3be54e61892b6a9ad5f546d0478R132